DANS-KNAW/easy-migration-tools
# DD-1116: find files in bagstore

Fixes DD-1116: find files in bagstore

# Description of changes

# How to test
with a running deasy VM

    cd ~/git/service/easy/easy-sword2-dans-examples
    ./run.sh Simple https://deasy.dans.knaw.nl/sword2/collection/1 IPDBSTest IPDBSTest src/main/resources/agreement-flow/valid/file-accessibilities
    cd ~/git/service/easy/easy-migration-tools
    poetry run list-bagstore-files -d data/uuids.txt > data/files.csv

Check also error logs for
* [x] inactive bags (response code 410)
* [x] bags that don't exist  (response code 404)

result:

```csv
uuid,doi,dataset-id,path,accessCategory
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/random images/image01.png,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/random images/image02.jpeg,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/random images/image03.jpeg,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/a/deeper/path/With some file.txt,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/accessibilities/anonymous.txt,ANONYMOUS
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/accessibilities/known.txt,KNOWN
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/accessibilities/request.txt,RESTRICTED_REQUEST
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/visibilities/anonymous.txt,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/visibilities/known.txt,
7f2b90d7-7e56-4ad4-a298-4474aa6c1938,10.80270/test-xztF-xeh-dxup,easy-dataset:32,data/visibilities/none.txt,
```


# Related PRs

replaces #3 

*

# Notify

@DANS-KNAW/dataversedans
